src/utils/mappers/CredentialMapper.ts

```diff
@@ -56,10 +57,8 @@ export function toCredentialSummary(verifiableCredential: ICredential, hash?: st
     ? new Date(verifiableCredential.expirationDate).valueOf() / 1000
     : 0
   const issueDate = new Date(verifiableCredential.issuanceDate).valueOf() / 1000
-  const credentialStatus =
-    !expirationDate || expirationDate >= new Date().valueOf() / 1000
-      ? CredentialStatusEnum.VALID
-      : CredentialStatusEnum.EXPIRED
+
+  const credentialStatus = getCredentialStatus(verifiableCredential)
   //TODO add revoked status support
```

Comment:
this comment can be moved to to isRevoked function, or can be removed now

Reply:
removed.